dspace/config/clarin-dspace.cfg

```diff
@@ -144,3 +144,12 @@ matomo.site.id = 1
 matomo.tracker.bitstream.site_id = 1
 matomo.tracker.oai.site_id = 1
 statistics.cache-server.uri = http://cache-server.none
+
+##### Citacepro config #####
+rest.properties.exposed= citace.pro.url
+citace.pro.url = https://www.citacepro.com/api/dspace/citace/oai
+citace.pro.url = dspace.tul.cz
+
+rest.properties.exposed= citace.pro.status
+# only use true or false for the value of citace pro status
+citace.pro.status = true
```

Comment:
For me as a developer this property name doesn't evokes that I can turn off/on the citace pro addon.  I think it would be better if you change it to something like `citace.pro.active` or `citace.pro.allowed` or something like that. What do you think?